src/Schema/SchemaInterface.php

```diff
@@ -390,9 +378,7 @@ public function getTableSchema(string $name, bool $refresh = false): TableSchema
      * @param bool $refresh Whether to fetch the latest available table schemas. If this is `false`, cached data may be
      * returned if available.
      *
-     * @return array The metadata for all tables in the database.
-     *
-     * @psalm-return list<TableSchemaInterface>
```

Comment:
Why remove `@psalm-return list<TableSchemaInterface>`? By code of `AbstractSchema` this method always returns list.